plugin/telegram.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""!
+    ____  ____  ______       __      __       __       _____
+   / __ )/ __ \/ ___/ |     / /___ _/ /______/ /_     |__  /
+  / __  / / / /\__ \| | /| / / __ `/ __/ ___/ __ \     /_ <
+ / /_/ / /_/ /___/ /| |/ |/ / /_/ / /_/ /__/ / / /   ___/ /
+/_____/\____//____/ |__/|__/\__,_/\__/\___/_/ /_/   /____/
+                German BOS Information Script
+                     by Bastian Schroll
+
+@file:        template_module.py
+@date:        14.01.2018
+@author:      Bastian Schroll
+@description: Template Plugin File
+"""
+import logging
+
+from telegram.error import (TelegramError, Unauthorized, BadRequest, TimedOut, ChatMigrated, NetworkError)
+from plugin.pluginBase import PluginBase
+import telegram
+import re
+import geocoder
+
+# ###################### #
+# Custom plugin includes #
+
+# ###################### #
+
+logging.debug("- %s loaded", __name__)
+
+
+class BoswatchPlugin(PluginBase):
+    """!Description of the Plugin"""
+
+    def __init__(self, config):
+        """!Do not change anything here!"""
+        super().__init__(__name__, config)  # you can access the config class on 'self.config'
+
+    def onLoad(self):
+        """!Called by import of the plugin"""
+        pass
+
+    def setup(self):
+        """!Called before alarm"""
+        self.bot = telegram.Bot(token=self.config.get("botToken", default=""))
+        pass
+
+    def fms(self, bwPacket):
```

Comment:
Nicht implementierte Typen können komplett entfernt werden, die Basisklasse wirft in diesem Fall sowieso eine entsprechende Log Meldung